gc3libs/utils.py

```diff
@@ -1926,13 +1712,13 @@ def __init__(self, initializer=None, **extra_args):
         if initializer is not None:
             try:
                 # initializer is `dict`-like?
-                for name, value in list(initializer.items()):
+                for name, value in initializer.items():
```

Comment:
There will probably be a lot of these things around -- `futurize` cannot distinguish when Py2's `.items()` is only used for iteration and when it's used for actually taking a copy :-(, so it defaults to the latter.

Reply:
Yes, there are many. Should I remove the list wrapping when I come across it? Only for cases of iteration?

Reply:
If you can clean up obvious cases, that would be helpful -- large-scale cleanup should probably be in a separate commit/PR.  Especially obnoxious are cases where a copy is being made for a purpose (e.g., mutating original list/dict) and code would become incorrect if we remove the wrapping...

Reply:
Yep, let's do separate PR one day that targets unnecessary `for k in list(dict.keys())`, i much just prefer just `for k in dict` :)